html-css/override-extend-variables/step2-instructions.md

```diff
@@ -1,32 +1,36 @@
-## Override PatternFly variables to achieve a new theme
-
-Now that we have a card on our page, let's customize the styles applied to it.
-
-1) <strong>Add a new color variable to use in the card.</strong> Let's make the links in the card purple! In PatternFly, all of the css values we use are first assigned to variables. We recommend doing the same when making customizations in your application.
+Now that you have a card on your page, let's customize the styles applied to it.
+ 
+1) <strong>Add a new color variable to use in the card.</strong> 
 
+Let's make the links in the card purple! In PatternFly, all of the css values you use are first assigned to variables. It is recommended doing the same when making customizations in your application.
+ 
 In `myapp.scss` inside of the `:root` block make a new variable name called `--my-app-card-theme--Color` and assign it to be purple: `#6200EE`.
-
+ 
 <strong>Hint:</strong> 
 ```
   --my-app-card-theme--Color: #6200EE;
 ```
+ 
+2) <strong>Override the global link color. </strong> 
 
-2) <strong>Override the global link color. </strong> In our current example of the card, the buttons in the card footer use the global link color. We can reassign the global link color variable to use our new custom color variable, so that all properties that use the global link color variable in the card will now use this color.
-
+In our current example of the card, the buttons in the card footer use the global link color. You can reassign the global link color variable to use our new custom color variable, so that all properties that use the global link color variable in the card will now use this color.
```

Comment:
```suggestion
In the current example of the card, the buttons in the card footer use the global link color. Reassign the global link color variable to use the new custom color variable so that all properties that use the global link color variable in the card will now use this color.
```